task-manager-backend/src/middlewares/is-admin-middleware.js

```diff
@@ -0,0 +1,13 @@
+const ApiError = require("../utils/api-error");
+
```

Comment:
we should not put spaces after every import. Lets apply these changes throughout the application.

Reply:
Okay, I added newline for constants also.

I will change this throughout the application.